### DISCUSSION TITLE
Get a diagram of the data model !

### DISCUSSION THREAD

**Discussion by drennvinn** [directus/directus#18825] [Feature Requests]
### Summary

It would be so nice if we could get a graphical overview of the data model (like a class diagram with its relationships) because when the project becomes substantial, it could relieve the eyes to be able to dive into it graphically.

### Motivation

...

### Detailed Design

...

### Requirements List

...

### Drawbacks

...

### Alternatives

...

### Adoption Strategy

...

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
